scripts/ci-test-summary.cjs

```diff
@@ -0,0 +1,130 @@
+const mocha = require('mocha');
```

Comment:
Is there a reason we specifically use CommonJS rather than ESM which we use everywhere else?

Reply:
I tried using ESM but kept getting errors, will update a screenshot here when I try it again